prime-router/src/main/kotlin/azure/ApiKeysFunctions.kt

```diff
@@ -169,24 +170,44 @@ SwIDAQAB
         )
     ]
 )
-@SecurityScheme(
-    name = "primeSecurity",
-    type = SecuritySchemeType.OAUTH2,
-    flows = OAuthFlows(
-        authorizationCode = OAuthFlow(
-            authorizationUrl = "https://hhs-prime.okta.com/oauth/authorize",
-            tokenUrl = "https://hhs-prime.okta.com/oauth/token",
-            scopes = [
-                OAuthScope(
-                    name = "org_admin",
-                    description = "Grants write access to single org"
-                ),
-                OAuthScope(name = "prime_admin", description = "Grants access to admin operations"),
-                OAuthScope(name = "user", description = "Grants read access")
-            ]
+
+@SecuritySchemes(
+    value = [
+        SecurityScheme(
+            name = "primeSecurity",
+            type = SecuritySchemeType.OAUTH2,
+            flows = OAuthFlows(
+                authorizationCode = OAuthFlow(
+                    authorizationUrl = "https://hhs-prime.okta.com/oauth/authorize",
+                    tokenUrl = "https://hhs-prime.okta.com/oauth/token",
+                    scopes = [
+                        OAuthScope(
+                            name = "org_admin",
+                            description = "Grants write access to single org"
+                        ),
+                        OAuthScope(name = "prime_admin", description = "Grants access to admin operations"),
+                        OAuthScope(name = "user", description = "Grants read access")
+                    ]
+                )
+            ),
+            description = "OAUTH2 Authroization for RS API Access."
+        ),
+        SecurityScheme(
+            name = "primeSecurityAPIKey",
+            type = SecuritySchemeType.APIKEY,
+            paramName = "x-functions-key",
+            description = "API Key Authorization."
```

Comment:
Maybe we should change this to say "Azure Function Key Authorization" or something instead to make it clear that this API key is associated with the specific azure function? Otherwise I think "API Key Authorization" is too generic and doesn't tell us a lot.